components/calendar/demo/calendar.html

```diff
@@ -62,8 +62,11 @@ <h2>Calendar (with events)</h2>
 						};
 
 						const calendar = document.querySelector('#eventsCalendar');
-						calendar.dayInfos = getEvents(getMinMaxDatesInView(calendar.selectedValue));
-						calendar.addEventListener('d2l-calendar-view-change', e => calendar.dayInfos = getEvents(e.detail));
+						calendar.dayInfos = getEvents(getMinMaxDatesInView(undefined, calendar.selectedValue));
+						calendar.addEventListener('d2l-calendar-view-change', e => {
+							console.log(e);
```

Comment:
Probably didn't want this left in here?

Reply:
Actually this was intentional to help anyone using this demo what the event details are. Happy to remove if anyone disagrees though.

Reply:
Ah ok cool, yep no concerns.